nemo/collections/common/tokenizers/aggregate_tokenizer.py

```diff
@@ -147,6 +151,85 @@ def ids_to_tokens(self, ids):
 
         return tokens
 
+    def ids_to_langs(self, ids):
```

Comment:
All of these methods are part of decoding - is it practical that user sees these methods ? Can we hide the internal ones or do we expect users to often use so many public methods

Reply:
addressed